src/frontend/components/Category/CategoryContainer.jsx

```diff
@@ -0,0 +1,40 @@
+import { Link } from "react-router-dom";
+import { CategoryCard } from "./CategoryCard";
+import "./Category.css";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
```

Comment:
It's a good practice & increases the readability of code if we separate the default import and the local imports

Example 

import { Link } from "react-router-dom";
import { useState, useEffect } from "react";
import axios from "axios";

import "./Category.css";
import { CategoryCard } from "./CategoryCard";